src/store/keyboard/context.js

```diff
@@ -0,0 +1,40 @@
+import { createContext, useContext, useState } from 'react';
+
+const KeyboardContext = createContext();
+
+export const KeyboardProvider = ({ children }) => {
+  const [isOptionKey, setIsOptionKey] = useState(false);
+
+  const initialvalue = {
+    isOptionKey,
+    setIsOptionKey,
+  };
+
+  return (
+    <KeyboardContext.Provider value={initialvalue}>
+      <div
+        role="button"
+        tabIndex={0}
+        onKeyDown={(e) => {
+          if (e.altKey) {
+            setIsOptionKey(true);
+          }
+        }}
+        onKeyUp={() => {
```

Comment:
event object `e` needs to be passed as the input parameter.

Reply:
thanks, it will be changed in the changed in the next commit